### ISSUE TITLE
BUG: Fix navbar of the open source programs category page.

### ISSUE THREAD

**Issue by Abidsyed25** [Sriparno08/Openpedia#285]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [ ] Codebase
- [X] User Interface
- [ ] Feature Request

## Description

The navbar of the open source program category page is not coded properly. It has only hamburger icon for all screensizes. I want to add the home, about, faq links on to the navbar and fix the hamburger icon to mobile view only.  I am attaching a below screenshot for reference. 

![navbar](https://github.com/Sriparno08/Openpedia/assets/116893970/288f1ab9-9f4d-4a7d-b6c9-208b549bf4fe)

**Comment by Sriparno08** [comment]
@Abidsyed25 Nice idea, but you're already assigned to another issue. Once its PR is merged, you will be considered for this issue. If anyone asks to be assigned to this issue before your PR is merged, he/she will be assigned.

**Comment by Sneha123-zudo** [comment]
I am intrested in this issue, please assign me this issue.

**Comment by antilneeraj** [comment]
@Sriparno08 Please assign this issue to me under JWOC.

**Comment by Abidsyed25** [comment]
Hey @Sriparno08, I have opened the issues after working on it. Assigning to others and they working on it may take time. Kindly give me permission to open the pull request. I can do it immediately.

**Comment by Sriparno08** [comment]
@Abidsyed25 Since you have already worked on this, we're assigning this issue to you. But from the next time, please start working after you are assigned to an issue. Otherwise, your hard work may not get counted.
